hack/run-openshift

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+#
+# Test the MongoDB image in OpenShift.
+#
+# IMAGE_NAME specifies a name of the candidate image used for testing.
+# The image has to be available before this script is executed.
+#
+
+set -exo nounset
+
+function prepare_oc_client() {
+  # Prepare oc command
+  if [[ ! -d ../openshift-client/ ]]; then
+    OC_RELEASE=$(curl -s https://api.github.com/repos/openshift/origin/releases | grep "browser_download_url" | grep "openshift-origin-client" | grep "linux-64bit" | sort | tail -n 1 | sed -e 's|"browser_download_url": ||' | tr -d ' "')
```

Comment:
yes, possibly because i was behind a proxy since the rate limiting is by IP.

getting the oc command isn't the problem, it's the call to the github api that you're using to determine what the latest release is that could be problematic if you're rate limited.

it's probably ok to leave it for now, but if you start seeing issues you may just want to hardcode the oc binary to download (and bump it as needed)